device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -164,14 +164,36 @@ void open() throws IOException
         }
     }
 
-    void registerMultiplexedDeviceClient(List<DeviceClientConfig> configs) throws InterruptedException
+    // Functionally the same as "open()", but without wrapping any thrown TransportException into an IOException
+    void multiplexingClientOpen() throws TransportException
     {
-        this.transport.registerMultiplexedDeviceClient(configs);
+        try
+        {
+            open();
+        }
+        catch (IOException e)
```

Comment:
We did this silly thing in the DeviceClient to work around the fact that we can't throw TransportException's directly in methods like ```deviceClient.open()``` becuase the open API existed before the TransportException did. To get around it, we just nested the meaningful exception into an IOException. I don't want to do the same thing for the multiplexing client though, so this code un-nests the exception when possible.

Reply:
Add that as a code comment, then please